java/Oleksii-Riabov/src/com/zoolatech/lecture1/tasks/_2/Task2.java

```diff
@@ -0,0 +1,50 @@
+package com.zoolatech.lecture1.tasks._2;
+
+import java.util.Scanner;
+
+/**
+ * Write a program that accepts a distance in meters and time in seconds
+ * and outputs a speed in meters/second and kilometers/hour. The result
+ * should be a floating point number.
+ */
+
+public class Task2 {
+
+    public static void main(String[] args) {
+
+        double distanceInMeters;
+        double timeInSeconds;
+
+        Scanner distanceScanner = new Scanner(System.in);
```

Comment:
actually, we can use a single instance of a scanner here

Reply:
Already remove redundant scanner instances in all tasks.